azure-template/tests/test_simple.py

```diff
@@ -0,0 +1,11 @@
+# the inclusion of the tests module is not meant to offer best practices for
+# testing in general, but rather to support the `find_packages` example in
+# setup.py that excludes installing the "tests" package
+from devtools_testutils import AzureMgmtTestCase
+
+class TemplateTest(AzureMgmtTestCase):
+    def setUp(self):
+        super(TemplateTest, self).setUp()
+
+    def test_case_default(self):
+        self.assertEqual(True, True)
```

Comment:
`assert True` or  `assert 2+2 == 4`. This is the recommended pytest syntax

Reply:
I now actually exercise a comparison between True and my package executing to return True.